pkg/client/application_profile.go

```diff
@@ -31,6 +31,30 @@ func (h *V1Client) GetApplicationProfileByName(profileName string) (*models.V1Ap
 
 	return nil, fmt.Errorf("Application profile '%s' not found.", profileName)
 }
+func (h *V1Client) GetApplicationProfileByNameAndVersion(profileName string, version string) (*models.V1AppProfileSummary, string, string, error) {
```

Comment:
could you please check if there is an API to return all versions of certain profile? we can reduce search complexity then.

Reply:
Within the application profile itself, it will return all the available versions of the profile and its own UID as an array of interfaces. but we cant query directly the particular version.